test/videojs-contrib-hls.test.js

```diff
@@ -652,6 +652,32 @@ QUnit.test('selects a playlist below the current bandwidth', function(assert) {
   assert.equal(this.player.tech_.hls.stats.bandwidth, 10, 'bandwidth set above');
 });
 
+QUnit.test('selects a primary rendtion when there are multiple rendtions share same attributes', function(assert) {
+  let playlist;
+
+  this.player.src({
+    src: 'manifest/master.m3u8',
+    type: 'application/vnd.apple.mpegurl'
+  });
+  openMediaSource(this.player, this.clock);
+  standardXHRResponse(this.requests[0]);
+
+  this.player.tech_.hls.playlists.master.playlists[0].attributes.BANDWIDTH = 528;
+  this.player.tech_.hls.playlists.master.playlists[1].attributes.BANDWIDTH = 528;
+  this.player.tech_.hls.playlists.master.playlists[2].attributes.BANDWIDTH = 728;
+  this.player.tech_.hls.playlists.master.playlists[3].attributes.BANDWIDTH = 728;
```

Comment:
the unit test above and also 'selects the correct rendition by tech dimensions' one covered playlists with same resolution and different bandwidths, and playlists with same bandwidths but different resolution, added the playlists that have the same resolution and bandwidths.